app/views/email_alerts/medical_safety_alerts/publication.html.erb

```diff
@@ -15,6 +15,12 @@
         <%= document_summary %>
       </p>
 
+      <% if updated_or_published == "updated" && document_change_note.present? %>
```

Comment:
It feels like it would be cleaner to just not populate `document_change_note` in the formatter if it shouldn't be shown, rather than putting the logic into the view